content/en/themes/creating.md

```diff
@@ -20,7 +20,11 @@ wip: true
 ---
 
 {{% warning "Use Absolute Links" %}}
-If you're creating a theme with plans to share it on the [Hugo Themes website](https://themes.gohugo.io/) please note that your theme's demo will be available in a sub-directory of website and for the theme's assets to load properly you will need to create absolute paths in the templates  by using either the [absURL](/functions/absurl) function or `.Permalink`. Also make sure not to use a forward slash `/` in the beginning of a `PATH`, because Hugo will turn it into a relative URL and the `absURL` function will have no effect.
+If you're creating a theme with plans to share it on the [Hugo Themes website](https://themes.gohugo.io/) please note the following: 
+- If you choose to use relative URLs in your template you need to quote them.
```

Comment:
This is a too broad statement, I suggest:

If you use relative URLs in links, they must be quoted, e.g `<a href="{{ "/blog" | relURL }}">` and `<img src="{{ "/images/logo.png"  | relURL }}">`


Or something like that.